site/content/docs/manifest/environment.en.md

```diff
@@ -167,9 +167,26 @@ List of [AWS Certificate Manager certificate](https://docs.aws.amazon.com/acm/la
 By attaching public or private certificates to your load balancer, you can associate your Backend Services with a domain name and reach them with HTTPS.
 See the [Developing/Domains](../developing/domain.en.md#use-domain-in-your-existing-validated-certificates) guide to learn more about how to redeploy services using [`http.alias`](./backend-service.en.md#http-alias).
 
-<span class="parent-field">http.private.</span><a id="http-private-subnets" href="#http-private-subnets" class="field">`subnets`</a> <span class="type">Array of Strings</span>  
+<span class="parent-field">http.private.</span><a id="http-private-subnets" href="#http-private-subnets" class="field">`subnets`</a> <span class="type">Array of Strings</span>   
 The subnet IDs to place the internal load balancer in.
 
+<span class="parent-field">http.private.</span><a id="http-private-security-groups" href="#http-private-security-groups" class="field">`security_groups`</a> <span class="type">Map</span>    
+Additional security groups to add to the internal load balancer.
```

Comment:
They are not necessarily "additional" - this field could be just configuring the sg that we put in front of the internal ALB. E.g. `security_groups.ingress.from_vpc` is adding the ingress, instead of adding an sg that allows that ingress (iirc).

Perhaps we can reword it to `Configure security groups to add to the internal load balancer.`?

Reply:
Done!